worth2/templates/pagetree/page.html

```diff
@@ -106,8 +106,17 @@
 {% endif %}
 
 {% if next_section %}
-<li class="next"><a href="{{next_section.get_absolute_url}}">{{next_section.label}} &rarr;</a></li>
 
+{% comment %}
```

Comment:
The next link should also be disabled if other pageblocks on the page are not "unlocked", e.g. for unanswered quiz questions.